tests/robot/_resources/test_data_sets/invalid_templates/minimal_persistent/persistent_minimal.composition.extdatetime.invalid.xml

```diff
@@ -44,6 +44,15 @@
     </composer>
     <content archetype_node_id="openEHR-EHR-OBSERVATION.minimal.v1" xsi:type="OBSERVATION">
       <!-- removed mandatory name -->
+      <archetype_details>
```

Comment:
@jakesmolka archetype_details should be optional https://specifications.openehr.org/releases/RM/Release-1.1.0/common.html#_overview_2